versioned_docs/version-0.9.1/Contributing/CTRLPanel Test.md

```diff
@@ -4,45 +4,48 @@ sidebar_position: 4
 
 :::info
 
-Please only use CTRLPanel installations that are not modifyed by plugins/modules/scripts or themes.
+Please only use Ctrlpanel installations that are not modifyed by plugins/modules/scripts or themes.
 
 :::
 
-This repository is using the playwright framework to provide E2E to help developers of the project to speed up development and provide a more stable and secure product.    
-In case you wanna help by adding some tests, playwright got a awsome documentaion [here](https://playwright.dev/docs/intro) but here are some basics we already took care of so you can start right away.  
+This repository is using the Playwright framework to provide E2E to help developers of the project speed up development and provide a more stable and secure product.    
+If you want to support and help us by adding tests, Playwright has a awesome documentation which you can find [here](https://playwright.dev/docs/intro), but below are some provided examples of basic Playwright testing incase you want to skip the documentation.  
   
 ## Setup  
 ### Requirements  
-- A setup version of CTRLPanel with the version you wanna test  
-- A admin account & a user account on this panel  
-- Basic knowlage of JS (or TS)
-- NodeJS 18+  
+- A version of Ctrlpanel that's already set up.
+- A administrator & client account on your dashboard
+- Basic knowledge of JS ([NodeJS](https://nodejs.org)) **OR** TS ([Typescript](https://www.typescriptlang.org/))
+- NodeJS v18 or newer
 
 ### Instructions
-1: clone [this](https://github.com/Ctrlpanel-gg/ctrlpanel-tests) repo  
-2: run `npm install` to install modules  
-3: run `npx playwright install` to install playwright browsers  
-4: rename .env.example to .env  
-5: populate .env with your data  
-6: you are ready to go  
+1: Clone [this](https://github.com/Ctrlpanel-gg/ctrlpanel-tests) GitHub repository
+2: Run `npm install` to install node modules  
+3: Run `npx playwright install` to install Playwright browsers  
+4: Rename `.env.example` to `.env`
+5: Change your `.env` with your dashboard's information  
+6: You're ready to begin testing!
   
 ## Testing  
 ### CLI  
-`´npx playwright test`  
+` npx playwright test `  
 ### UI  
-`npx playwright test --ui`  
+` npx playwright test --ui `  
+
+## Start your Tests
+### Simple Rules
+Because you need to be logged in for almost everything, we already took care of this. Playwright tests are loaded by regex matches.    
```

Comment:
this line seems weird, ill try to pinpoint it later

Reply:
i didnt understand it at all as well so i just left it alone